boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -40,8 +41,12 @@ class FsmStateMachineTest : public FsmTest
         heartbeat_monitor = App_SharedHeartbeatMonitor_Create(
             get_current_ms, DEFAULT_HEARTBEAT_TIMEOUT_PERIOD_MS,
             DEFAULT_HEARTBEAT_BOARDS_TO_CHECK, heartbeat_timeout_callback);
+        primary_flow_meter   = App_FlowMeter_Create(get_flow_rate);
+        secondary_flow_meter = App_FlowMeter_Create(get_flow_rate);
```

Comment:
Did you mean to use the same fake function for primary and secondary flow meters? I think having `get_primary_flow_rate` and `get_secondary_flow_rate` would be more clear

Reply:
Yeah that would make sense since they are separate fxns IRL. Fixed.